### PR TITLE
JBIDE-22601 - fix regression causing error starting rse.mgmt profile …

### PR DESCRIPTION
**Pull request by robstryker**
…server

### CODE REVIEW

File: as/plugins/org.jboss.ide.eclipse.as.core/jbosscore/org/jboss/ide/eclipse/as/core/util/ServerHomeValidationUtility.java [hunk 1]
```diff
@@ -49,23 +48,27 @@ public IStatus validateServerHome(IServer server, boolean alwaysThrow) throws Co
...
-		IStatus s = validateServerHome(server, det, opts, fs);
+		IStatus s = validateServerHome(server, det, fs);
 		if( alwaysThrow && !s.isOK()) {
 			throw new CoreException(s);
 		}
 		return s;
 	}
 	
 	public IStatus validateServerHome(IServer server, IServerDetailsController details, 
-			IDeploymentOptionsController options, IFilesystemController fs) throws CoreException {
```
Comment (alexeykazakov) [inline as/plugins/org.jboss.ide.eclipse.as.core/jbosscore/org/jboss/ide/eclipse/as/core/util/ServerHomeValidationUtility.java]: You are breaking API of a public class/method here. Do we really have to do it? Why not deprecate the old method and add a new one. You can ignore options here.

Comment (robstryker) [inline as/plugins/org.jboss.ide.eclipse.as.core/jbosscore/org/jboss/ide/eclipse/as/core/util/ServerHomeValidationUtility.java]: the class is only a month old and nobody else uses it yet.  In fact the introduction of this class is what broke the code in the first place.

Comment (alexeykazakov) [inline as/plugins/org.jboss.ide.eclipse.as.core/jbosscore/org/jboss/ide/eclipse/as/core/util/ServerHomeValidationUtility.java]: OK. thanks

**Comment by alexeykazakov** [comment]
LGTM.

**Comment by robstryker** [comment]
pushed to master and respin-b

**Pull request merged**